### PR TITLE
[ADD] website_event_baidu_map

### PR DESCRIPTION
**Pull request by martintorresvm**

### CODE REVIEW

**Comment by martintorresvm** [comment]
Please review @elicoidal

**Comment by elicoidal** [comment]
@victormartinelicocorp Can you add travis and coverall for this repo?

File: website_event_baidu_map/README.rst [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+ * have basic modules installed (website_event)
+
+
+Configuration
+
+=============
+
+To configure this module, you need to:
+
+* Go to Settings/Parameters/System Parameters and create a record which Key must be named baidu.api and the Value like [http://api.map.baidu.com/api]?v=[2.0]&ak=[your ak].
```
Comment (elicoidal) [inline website_event_baidu_map/README.rst:26]: use enumerated list better

File: website_event_baidu_map/README.rst [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+* Rona Lin <rona.lin@elico-corp.com>
+
+Maintainer
+----------
+
+.. image:: https://www.elico-corp.com/logo.png
+:alt: Elico Corp
+:target: https://www.elico-corp.com
+
+This module is maintained by Elico Corporation.
```
Comment (elicoidal) [inline website_event_baidu_map/README.rst:45]: use OCA template

File: website_event_baidu_map/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+{
+    'name': 'Website Event Baidu Map',
+    'version': '9.0.1.0.1',
+    'category': 'Website',
+    'depends': [
+        'website_event',
+    ],
+    'author': 'Elico Corp, Odoo Community Association (OCA)',
+    'license': 'AGPL-3',
+    'website': 'https://www.elico-corp.com, https://odoo-community.org',
```
Comment (elicoidal) [inline website_event_baidu_map/__openerp__.py:13]: remove second website

File: website_event_baidu_map/__openerp__.py [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+    'data': [
```
Comment (elicoidal) [inline website_event_baidu_map/__openerp__.py:14]: add support key

File: website_event_baidu_map/README.rst [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
+
+Configuration
+
+=============
+
+To configure this module, you need to:
+
+* Go to Settings/Parameters/System Parameters and create a record which Key must be named baidu.api and the Value like [http://api.map.baidu.com/api]?v=[2.0]&ak=[your ak].
+
```
Comment (elicoidal) [inline website_event_baidu_map/README.rst:27]: Add Runbot and Issue section from standard OCA template

**Review by elicoidal — CHANGES_REQUESTED** [review]
mostly some formal changes

**Comment by elicoidal** [comment]
@victormartinelicocorp I think this one could be finished easily

**Comment by martintorresvm** [comment]
v9 is old version

**Pull request closed without merging**